roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -105,6 +105,31 @@ def get_var(self, *keys, **kwargs):
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
         return value
 
+    def find_ansible_mount(self, path):
+        """Return the mount point for path from ansible_mounts."""
+
+        # reorganize list of mounts into dict by path
+        mount_for_path = {
+            mount['mount']: mount
+            for mount
+            in self.get_var('ansible_mounts')
+        }
+
+        mount_point = path
+        while mount_point not in mount_for_path:
+            if mount_point in ["/", ""]:  # "/" not in ansible_mounts???
```

Comment:
incredulity that `ansible_mounts` would ever come back without `/` mapped :) i guess it's not a very illuminating comment.